README.md

```diff
@@ -84,16 +90,16 @@ Add `image-sequencer` to your list of dependencies and run `npm install`
 
 Globally install Image Sequencer:
 
-```
-$ npm install image-sequencer -g
+```sh
+npm install image-sequencer -g
```

Comment:
Shouldn't we change this to `yarn global add image-sequencer`??

Reply:
I will change that later. Currently I would like to fix the errors and warnings. The deps have to be updates. Thanks!